LR scheduler:  Error: 'Sine' object has no attribute '_step_count'

Epoch 1/22, Steps:   0%|                                        | 29/30000 [01:08<17:47:11,  2.14s/it, lr=0, step_loss=0.245]2024-11-10 19:28:47,024 [ERROR] Failed to get the last learning rate from the scheduler. Error: 'Sine' object has no attribute '_step_count'
Epoch 1/22, Steps:   0%|                                        | 30/30000 [01:11<18:10:36,  2.18s/it, lr=0, step_loss=0.023]2024-11-10 19:28:49,022 [ERROR] Failed to get the last learning rate from the scheduler. Error: 'Sine' object has no attribute '_step_count'
Epoch 1/22, Steps:   0%|                                        | 31/30000 [01:13<17:42:51,  2.13s/it, lr=0, step_loss=0.118]2024-11-10 19:28:51,085 [ERROR] Failed to get the last learning rate from the scheduler. Error: 'Sine' object has no attribute '_step_count'
Epoch 1/22, Steps:   0%|                                       | 32/30000 [01:14<19:15:41,  2.31s/it, lr=0, step_loss=0.0314]^C

yeah, i tried that before. but all differnt LR scheduler would give same error:
Epoch 1/22, Steps: 0%| | 29/30000 [01:08<17:47:11, 2.14s/it, lr=0, step_loss=0.245]2024-11-10 19:28:47,024 [ERROR] Failed to get the last learning rate from the scheduler. Error: 'Sine' object has no attribute '_step_count'

specifically, i tried to set: "LR_SCHEDULER=constant" in config.env

--- a/helpers/training/custom_schedule.py
+++ b/helpers/training/custom_schedule.py
@@ -452,6 +452,7 @@ class Sine(LRScheduler):
         self.verbose = verbose
         self._last_lr = self.base_lrs
         self.total_steps = 0  # Track total steps for a continuous wave
+        super().__init__(optimizer, last_step, verbose)
 
     def get_lr(self):
         # Calculate learning rates using a continuous sine function based on total steps

